api/src/main/resources/org/openmrs/module/mirebalaisreports/sql/mexicoCesMeds.sql

```diff
@@ -0,0 +1,37 @@
+SELECT
+    MAX(d.name) "Medicamento",
```

Comment:
I don't think you need the "MAX" function here, and I'm not sure what it's purpose is.  You are already grouping on drug.